File: src/test/java/com/roscopeco/jasm/model/IfTests.java
package com.roscopeco.jasm.model;

public interface IfTests {
    boolean testEqWhenEqualPasses();
    boolean testEqNotEqualPasses();
    boolean testGeWhenLessPasses();
    boolean testGeWhenEqualPasses();
    boolean testGeWhenGreaterPasses();
    boolean testGtWhenLessPasses();
    boolean testGtWhenGreaterPasses();
    boolean testLeWhenLessPasses();
    boolean testLeWhenEqualPasses();
    boolean testLeWhenGreaterPasses();
    boolean testLtWhenLessPasses();
    boolean testLtWhenGreaterPasses();
    boolean testNeWhenEqualPasses();
    boolean testNeNotEqualPasses();
}

File: src/test/kotlin/com/roscopeco/jasm/model/IfTests.kt
package com.roscopeco.jasm.model

interface IfTests {
    fun testEqWhenEqualPasses(): Boolean
    fun testEqNotEqualPasses(): Boolean
    fun testGeWhenLessPasses(): Boolean
    fun testGeWhenEqualPasses(): Boolean
    fun testGeWhenGreaterPasses(): Boolean
    fun testGtWhenLessPasses(): Boolean
    fun testGtWhenGreaterPasses(): Boolean
    fun testLeWhenLessPasses(): Boolean
    fun testLeWhenEqualPasses(): Boolean
    fun testLeWhenGreaterPasses(): Boolean
    fun testLtWhenLessPasses(): Boolean
    fun testLtWhenGreaterPasses(): Boolean
    fun testNeWhenEqualPasses(): Boolean
    fun testNeNotEqualPasses(): Boolean
}